app/controllers/concerns/users/filterable.rb

```diff
@@ -46,7 +46,7 @@ def filter_users_by_orientation_type
              .joins(orientations: :orientation_type)
              .where(orientation_types: { name: params[:orientation_type] })
              .where(orientations: { organisations: { department_id: current_department_id } })
-             .where("orientations.starts_at <= ?", Time.zone.now)
+             .where(orientations: { starts_at: ..Time.zone.now })
```

Comment:
Je suis partagé 😅

Reply:
Pourquoi ici tu mets 2 points et en dessous 3 ?

Reply:
Oui j'avoue c'est un peu perturbant au début ça me l'a fait aussi :D. 
Après la syntaxe est plus simple (pas besoin de point d'interrogations etc.) et plus ruby. Je me dis que peut-être ça vaut le coup de voir si on arrive à s'habituer à cette nouvelle syntaxe, et au pire on pourra rechanger 🤷‍♂️ .

> Pourquoi ici tu mets 2 points et en dessous 3 ?

3 points c'est pour les inégalités strictes et 2 points ça inclut la valeur en question

Reply:
J'aime bcp cette nouvelle syntaxe pour ma part